core/services/src/main/java/org/exoplatform/services/attachments/model/AttachmentsEntityType.java

```diff
@@ -0,0 +1,5 @@
+package org.exoplatform.services.attachments.model;
+
+public enum AttachmentsEntityType {
+  WIKI, EVENT, TASK;
```

Comment:
Attachment types should be extensible through a component plugin.
We should be able to attach files to any kind of applications without having to change code provided in ecms.

Reply:
example

> https://github.com/Meeds-io/social/blob/develop/webapp/portlet/src/main/webapp/spaces-navigation/initComponents.js#L16-L24
> https://github.com/Meeds-io/gatein-portal/blob/develop/portlet/web/src/main/webapp/vue-apps/hamburger-menu/components/ExoHamburgerMenuNavigation.vue#L129